pkg/config/parameters.go

```diff
@@ -182,8 +182,7 @@ func (n NamespacedName) Validate() error {
 
 // TLSParameters holds configuration file TLS configuration details.
 type TLSParameters struct {
-	MinimumProtocolVersion string `yaml:"minimum-protocol-version"`
-	MaximumProtocolVersion string `yaml:"maximum-protocol-version"`
+	ProtocolParameters ProtocolParameters `yaml:"protocol"`
```

Comment:
We'll need to embed the new struct here so we don't make a breaking change to the config file YAML representation:
```suggestion
	ProtocolParameters
```

Reply:
hmmm I see. I almost did that too lol but went for adding a new field instead